test/system/projects/samples_test.rb

```diff
@@ -1298,19 +1299,6 @@ class SamplesTest < ApplicationSystemTestCase
         assert_selector 'tr:last-child td:nth-child(2)', text: @sample2.name
       end
 
-      click_on 'metadatafield2'
```

Comment:
Any specific reason of why this section was removed?

Reply:
Because it was redundant for the test and was causing issues due to scrolling and sticky columns. If you can sort one column you can sort any column.